client/checkout/woopay/express-button/woopay-express-checkout-button.js

```diff
@@ -46,6 +50,20 @@ export const WoopayExpressCheckoutButton = ( {
 		isAddToCartDisabled,
 	} = useExpressCheckoutProductHandler( api, isProductPage );
 
+	let sessionDataPromise = null;
+	if ( ! isProductPage ) {
```

Comment:
At the moment, the product page is not supported. Mainly because at this point we are not able to determine whether the item on the product page will be purchased or not. Therefore, in this case, we fall back to the previous approach.

Reply:
After refactoring, this is no longer an issue. Resolving conversation.